epictrack-web/src/components/myWorkplans/Card/CardBody.tsx

```diff
@@ -43,20 +43,21 @@ const CardBody = ({ workplan }: CardProps) => {
     >
       <Grid item container spacing={2}>
         <Grid item xs={9}>
-          <ETHeading4
-            bold
-            color={Palette.neutral.dark}
-            xs
-            sx={{
-              overflow: "hidden",
-              textOverflow: "ellipsis",
-              whiteSpace: "nowrap",
-            }}
-          >
-            <Tooltip title={workTitle} placement="top-start">
-              <span>{workTitle}</span>
-            </Tooltip>
-          </ETHeading4>
+          <Tooltip title={workTitle} placement="top-start">
```

Comment:
Did you check the tooltip. Is it working. As I understand the Tooltip component work only if it wraps a built in component not ETHeading4 which is custom component. Also, please check the way we have implemented enableTooltiop and enableEllipsis stuff in other components in shared folder

Reply:
good idea i added the tooltop to enable the tool tip and ellipsis